mrmustard/lab_dev/wires.py

```diff
@@ -331,12 +342,17 @@ def _outin(self_in: int, self_out: int, other_in: int, other_out: int) -> np.nda
         else:  # no wires on other
             return np.array([self_out, self_in], dtype=np.int64)
 
-    def __rshift__(self, other: Wires) -> Wires:
+    def __matmul__(self, other: Wires) -> Wires:
         r"""
-        A new Wires object with the wires of ``self`` and ``other`` combined as two
-        components in a circuit: the output of self connects to the input of other wherever
-        they match. All surviving wires are arranged in the standard order.
-        A ValueError is raised if there are any surviving wires that overlap.
+        A new ``Wires`` object with the wires of ``self`` and ``other`` combined.
+
+        The output of ``self`` connects to the input of ``other`` wherever they match. All
+        surviving wires are arranged in the standard order.
+
+        This function does not add missing adjoints.
```

Comment:
```suggestion
        This function only performs the contractions given and it does not add missing adjoints.
```

A small suggestion to be more clear on this.